boards/efm32g-st700/include/board.h

```diff
@@ -0,0 +1,65 @@
+
+
```

Comment:
Just to avoid any possible misunderstandings: of course you don't have to grant the copyright to Freie Universität Berlin. ;-)